### ISSUE TITLE
failed private pkg still restricts on version number

### ISSUE THREAD

**Issue by ashleygwilliams** [npm/registry-issue-archive#12]
## tl;dr

**expected:** if i accidentally try to publish privately at one version number and it fails because i dont have permissions, i should be able to publish at that version publicly

**what happens instead:** while publishing a private scoped package @ version 1.0 fails, an attempt to publish with `--access=public` @ version 1.0 fails with error ""You cannot publish over the previously published version 1.0.1." : private-to-public-scoped".
## steps to reproduce
1. fork and clone https://github.com/ashleygwilliams/is-npm-working
2. `cd is-npm-working`
3. `./test.sh`

the third test will demo the 403 error

**Comment by joshmanders** [comment]
Thank you for this ticket, @ashleygwilliams!

**Comment by nexdrew** [comment]
Looks like this was fixed somewhere along the way. @ashleygwilliams and I have both verified you can publish the same version of a username-scoped package with `npm publish --access=public` after an initial `npm publish` (private) fails with a 402.

@joshmanders Please retest when you get a chance and let us know what you find. Thanks.
